src/app/components/lineage-graph/lineage-graph-heigth.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+    providedIn: "root",
+})
+export class LineageGraphHeigthService {
+    private componentsPaddings = 90;
```

Comment:
If this is a constant, it has to be readonly and UPPER_CASE. But where does the value come from? Can it be computed on-the-floy as well?

Reply:
Fixed